test/tests_for_sdkmr/ee2_SDKMethodRunner_test.py

```diff
@@ -283,22 +276,18 @@ def test_cancel_job2(self, rq_mock, condor_mock):
         runner.get_condor = MagicMock(return_value=condor_mock)
         fixed_rj = EE2RunJob(runner)
         fixed_rj._get_module_git_commit = MagicMock(return_value="hash_goes_here")
-        fixed_rj.sdkmr.catalog_utils.list_client_group_configs = MagicMock(
-            return_value="cg goes her"
-        )
 
         runner.get_runjob = MagicMock(return_value=fixed_rj)
 
         # ctx = {"user_id": self.user_id, "wsid": self.ws_id, "token": self.token}
         job = get_example_job().to_mongo().to_dict()
-        job["method"] = job["job_input"]["app_id"]
+        job["method"] = job["job_input"]["method"]
         job["app_id"] = job["job_input"]["app_id"]
         job["service_ver"] = job["job_input"]["service_ver"]
 
         si = SubmissionInfo(clusterid="test", submit=job, error=None)
 
         condor_mock.run_job = MagicMock(return_value=si)
-        condor_mock.extract_resources = MagicMock(return_value=self.cr)
```

Comment:
Another example of mocks for deleted methods that can be removed without causing the test to fail. There are a lot like this scattered through the tests.